src/org/opendatakit/briefcase/ui/MessageStrings.java

```diff
@@ -80,4 +80,6 @@
   public static final String PROXY_SET_ADVICE =
       "If you are behind a proxy, try setting up your proxy details through 'Settings' tab.";
 
+  public static final String PARALLEL_DOWNLOADS = "Use Parallel Downloads";
```

Comment:
Ok, I thought download implied that, but it's the only place where the word download exists in the UI. I will reword this to make it more clear.